README.md

```diff
@@ -156,23 +156,32 @@ _Not tested by the current maintainers_
+
+#### Requirements for a raster file
+
+1. The format must be GeoTIFF, with a `.tif` file extension.
+2. Band 0 must contain a unique identifier for each set of text to display.  If the file contains multiple bands, all but the first will be ignored.
+3. Band 0 must contain unsigned integers no larger than 254 (they'll be stored as a single byte, real numbers will be rounded off to the nearest integer, and 255 is reserved for NODATA).
+4. The values in Band 0 must be higher for more serious warnings.  This is because the file will be reprojected to EPSG:4326 during import, and values are rounded up in the reprojection.  As long as higher values are the more serious warnings, this will not create a risk of people seeing an inappropriate "all clear" message for their location.
+5. Each data source must be one single `.tif` file.  Large files will be tiled automatically during the load but the data import pipeline does not currently have the ability to combine rasters.
 
 ### Fully automated pipeline
 
-If the structure of your text content is simple enough, you can import shapefiles and snuggets automatically without having to do much manual work. We recommend using this pathway if possible, because it makes moving the site to a new server significantly easier. To do this, you will need a `snuggets.csv` file with the same columns as the example one we've included in `data.zip`. The columns can be in any order, but the headings must be exactly as typed here:
+If the structure of your text content is simple enough, you can import shapefiles/rasters and snuggets automatically without having to do much manual work. We recommend using this pathway if possible, because it makes moving the site to a new server significantly easier. To do this, you will need a `snuggets.xlsx` file with the same columns as the example one we've included in `data.zip`. The columns can be in any order, but the headings must be exactly as typed here:
 
-- `heading` : A human-readable heading that describes the content of this shapefile, to be displayed on the page. This will correspond to the shapefile group.
+- `heading` : A human-readable heading that describes the content of this shapefile or raster, to be displayed on the page. This will correspond to the shapefile group.
 - `section` : A section name that will be displayed on the page (must not be empty)
-- `shapefile` : The file name for the shapefile this row corresponds to, without the extenstion. For example: `EQ_GroundShaking_MostLike` for text that relates to the content of `EQ_GroundShaking_MostLike.shp`. (must not be empty; must correspond exactly to the available shapefiles)
+- `shapefile` : The file name for the shapefile or raster file this row corresponds to, without the extention. For example: `EQ_GroundShaking_MostLike` for text that relates to the content of `EQ_GroundShaking_MostLike.shp`. (must not be empty; must correspond exactly to the available files)
```

Comment:
'extension' i think

Reply:
lol you're right.  I "fixed" that typo to another typo.